libs/reglibs.go

```diff
@@ -0,0 +1,69 @@
+package libs
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"strings"
+)
+
+func init() {
+	RegisterLibraries()
+}
+
+//go:embed includes/*
+var efs embed.FS
+
+func RegisterLibraries() {
```

Comment:
Should be unexported, e.g. `registerLibraries`. In which case it can be just inlined into `init` itself.